### DISCUSSION TITLE
Default component/UI Figma file

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#7142] [Help] [answered]
Hi all, 

Does anyone know if there is a Figma file containing mock-ups of all the default UI and components used in Budibase?

I'm currently building out a client-facing portal, and as part of the design process I'd like to share some mock-ups for how the app will look. 

Thanks!

**Comment by AndyWhann** [comment, marked as answer]
Hi @adamjdavies 

We use the Adobe Spectrum design system for all of our default components.

To create mock ups or designs with the Spectrum components you would need to use the Plugin for Adobe XD as they unfortunately don't offer Figma files.

https://spectrum.adobe.com/page/spectrum-xd-plugin/

**Reply by ghost** [reply]
Hi @AndyWhann 

Thanks for getting back to me. 

Ah, no worries then. Will have to create my own version in Figma.
